### PR TITLE
Added the security.keytab.path property to cdap-default

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -2631,6 +2631,15 @@
   </property>
 
   <property>
+    <name>security.keytab.path</name>
+    <value></value>
+    <description>
+      Configuration for specifying the location of kerberos keytabs. The location can contain ${name}
```
Comment (dereklwood) [inline cdap-common/src/main/resources/cdap-default.xml]: maybe make it more clear these keytabs are for impersonation?  I can imagine somebody putting the path of ``cdap.master.kerberos.keytab`` here

File: cdap-common/src/main/resources/cdap-default.xml [hunk 2]
```diff
@@ -2631,6 +2631,15 @@
   </property>
 
   <property>
+    <name>security.keytab.path</name>
+    <value></value>
+    <description>
+      Configuration for specifying the location of kerberos keytabs used for impersonation.
+      The location can contain ${name} which will be replaced by the user/owner of the entities name.
```
Comment (anew) [inline cdap-common/src/main/resources/cdap-default.xml]: Location of .... replaced by the short user name of the principal being impersonated.

**Comment by Nishith** [comment]
Updated the description, please take another look.

**Comment by anew** [comment]
LGTM

**Pull request merged**